docs/installation/frameworks/vuejs-v2.md

```diff
@@ -238,13 +238,59 @@ Finally, import the build in your application:
 
 Integrating the rich text editor from source allows you to use the full power of {@link framework/index CKEditor 5 Framework}.
 
+### Vite
+
+You need to install some packages to use CKEditor from source with Vue 2 and Vite: the official CKEditor Vue 2 component, the CKEditor Vite plugin, and a theme. Also, Vite uses Vue 3 by default, so keep that in mind if you want to start from scratch. This guide assumes you have Vue 2 with Vite already configured.
```

Comment:
```suggestion
This guide assumes you have Vue 2 with Vite already configured. `create-vue` is using Vue 3 by default, so keep that in mind if you want to start from scratch. 

You need to install some packages to use CKEditor from source with Vue 2 and Vite: the official CKEditor Vue 2 component and the CKEditor Vite plugin.`
```

We do not install the theme in the next command. Also, Vite as a bundler is framework agnostic, so it's better to mention `create-vue` instead of stating that Vite is using Vue 3 by default. I've added my suggestion, but feel free to rewrite it if needed.

Reply:
True, Vite is not exclusive to Vue. I think the paragraph is ok, so I'll commit it, and you can take a look.